CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # CHANGELOG
 
+## v0.4.0
+
+### Summary
+  This version supports EKS-Fargate pods and bumps the default envoy version to v1.12.2
+
+### Changes:
+* Copy attribution doc to image (#118, @nckturner)
```

Comment:
Can you just make these links to be consistent with the rest of the changelog?  If its annoying just write a script to generate it :D

Reply:
:D